lib/std/bounded_array.zig

```diff
@@ -9,7 +9,7 @@ const testing = std.testing;
 /// runtime, but whose maximum size is known at comptime, without requiring
 /// an `Allocator`.
 ///
-/// ```zig
+/// ```
```

Comment:
All markdown code blocks should be fully deleted in favor of doc tests.

Reply:
#16472 already seems to be working on this so I'll just leave this as-is.

Reply:
Sorry, I'll get around to getting that to be mergable soon.